eth-contracts/migrations/7_governance_migration.js

```diff
@@ -0,0 +1,20 @@
+const Registry = artifacts.require('Registry')
+const Governance = artifacts.require('Governance')
+
+const ownedUpgradeabilityProxyKey = web3.utils.utf8ToHex('OwnedUpgradeabilityProxy')
+
+const VotingPeriod = 10 // blocks
```

Comment:
TODO - i'll revert this to 48hrs

Reply:
is this required at all if you hard code the default to 11520?

Reply:
I removed the default val and am just going to pass it through constructor. I think that's a fine pattern anyway, but also makes it much easier to test haha. Otherwise would have to advance 11520 blocks in each test